apps/src/lib/client/signing.rs

```diff
@@ -174,7 +175,7 @@ pub async fn sign_wrapper(
     let tx = {
         WrapperTx::new(
             Fee {
-                amount: args.fee_amount,
+                amount: Amount::from(100),
```

Comment:
let's add a `const MIN_FEE` for this somewhere in core and re-use it in other places